pkg/cli/admin/release/release.go

```diff
@@ -26,3 +30,29 @@ func NewCmd(f kcmdutil.Factory, streams genericclioptions.IOStreams) *cobra.Comm
 	cmd.AddCommand(NewMirror(f, streams))
 	return cmd
 }
+
+// Bind adds the options to the flag set.
+func (o *ArchOptions) Bind(flags *pflag.FlagSet) {
+	flags.StringVar(&o.Arch, "arch", o.Arch, "Specify a release's architecture (amd64|arm64|ppc64le|s390x). When used, it will bypass searching the current connected cluster for the release payload.")
+}
+
+type ArchOptions struct {
+	Arch string
+}
+
+// Validating Arch, if --arch is passed
+func (o *ArchOptions) Validate() error {
+
+	if len(o.Arch) > 0 {
+
+		o.Arch = strings.ReplaceAll(o.Arch, "x86_64", "amd64")
+		o.Arch = strings.ReplaceAll(o.Arch, "aarch64", "arm64")
+
+		switch o.Arch {
+		case "", "amd64", "arm64", "ppc64le", "s390x":
+			return nil
+		}
+		return fmt.Errorf("%s is not a valid arch", o.Arch)
```

Comment:
I don't think we need to vet these before making the update service request.  We can just report "release `${VERSION}` not found in `${URI}`" or "no releases found in `${URI}`" as hints that maybe the update service on the other end doesn't like the query parameters.  Failing fast doesn't win much, and blocking access to new architectures that the update service might know about seems risky.